Assets/Scripts/Player/Input Parser/InputParser.cs

```diff
@@ -27,13 +32,31 @@ private void OnDisable()
         private void AddListeners()
         {
             _inputActionAsset["MoveTowards"].performed += MoveTowards;
+            _inputActionAsset["Interact"].performed += Interact;
         }
 
         private void RemoveListeners()
         {
             _inputActionAsset["MoveTowards"].performed -= MoveTowards;
+            _inputActionAsset["Interact"].performed -= Interact;
         }
 
         private void MoveTowards(InputAction.CallbackContext context) => _playerMovement.StartMovingEvent();
+
+        private void Interact(InputAction.CallbackContext context)
+        {
+            Ray ray = _mainCamera.ScreenPointToRay(SetMousePos());
+            Physics.Raycast(ray.origin, ray.direction *10, out RaycastHit hit);
```

Comment:
```suggestion
            Physics.Raycast(ray.origin, ray.direction * interactableRayDistance, out RaycastHit hit);
```
10 is hardcoded, hier graag een [SerializeField, Range()] private float.